wfe2/wfe.go

```diff
@@ -60,6 +60,12 @@ const (
 	newOrderPath      = "/acme/new-order"
 	orderPath         = "/acme/order/"
 	finalizeOrderPath = "/acme/finalize/"
+
+	getAPIPrefix       = "/get/"
+	getOrderPath       = getAPIPrefix + "order/"
+	getAuthzv2Path     = getAPIPrefix + "authz-v3/"
```

Comment:
Given the -v3 thing doesn't really mean anything anymore, how would you feel about axing it for these? It wouldn't match the /acme/ endpoints, but not sure how confusing that'd actually be.

Reply:
:laughing: I actually left myself a TODO comment to drop the "-v3" thing for these APIs and then changed my mind in https://github.com/letsencrypt/boulder/pull/4645/commits/e45f392909791e46e0b38314bc2426e598637c41

I thought it would be slightly easier for debugging if we matched the paths to the /acme/ endpoints. One less thing to mess about with when copy/pasting the ACME URLs.

I don't feel strongly about it if you do.

Reply:
:+1: works for me.